workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionCard/AccountSubscriptionCard.jsx

```diff
@@ -50,13 +52,28 @@ const AccountSubscriptionCard = ({
 		[accountSubscriptionUsageData]
 	);
 
+	let quantity = 0;
+
+    const now = new Date();
```

Comment:
@amosfong I am noticing that this file has a mixture of spaces and tabs, not sure how this is happening, as the sf should catch that, I may look into why this is being missed, maybe a regression or something

Reply:
ah it's because SF doesnt format .jsx. That's one of the reasons to move to tsx. I'll fix the spaces